GitSourceControl.uplugin

```diff
@@ -16,7 +16,7 @@
 	"Modules": [
 		{
 			"Name": "GitSourceControl",
-			"Type": "UncookedOnly",
+			"Type": "Editor",
```

Comment:
To my knowledge `UncookedOnly` is preferred

Reply:
Do we really want to interact with git anyhow outside of the editor?

Reply:
Yes, there's an expectation that source control module is available outside of the editor in many Unreal tools from Epic

Reply:
just out of curiosity, could you give me an example because I couldn't find any in the engine folder